manifests/m/Microsoft/NetMon/3.4/Microsoft.NetMon.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.NetMon
+PackageVersion: 3.4
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download.microsoft.com/download/7/1/0/7105C7FF-768E-4472-AFD5-F29108D1E383/NM34_x64.exe
+  InstallerSha256: 3A443055A478384DDD184C39A7B1ACEA9F213719D26E93204F782CB14DFB562A
+  InstallerSwitches:
+    Silent: /q
+    SilentWithProgress: /q
```

```suggestion
  InstallerSwitches:
    Silent: /q
    SilentWithProgress: /q
  AppsAndFeaturesEntries:
    - DisplayVersion: 3.4.2350.0
      ProductCode: '{8C5B5A11-CBF8-451B-B201-77FAB0D0B77D}'
```
To match the version in Registry / Control Panel.